app/views/includes/omic-organisation-overview.njk

```diff
@@ -22,7 +22,7 @@
             <th class="govuk-table__cell" headers="name om">Total/ Average</th>
             <th class="govuk-table__cell" headers="cases total">{{ overviewDetails.totals.totalTotalCases }}</th>
             <th class="govuk-table__cell" headers="custody total">{{ overviewDetails.totals.totalCustodyPoints }}</th>
-            <th class="govuk-table__cell" headers="licence total">{{ overviewDetails.totals.totalLicencePoints }}</th>
+            <th class="govuk-table__cell" headers="licence total">{{ overviewDetails.totals.totalLicensePoints }}</th>
```

Comment:
Same as above

Reply:
changed the id's by mistake in this file